bench_cli/configuration.py

```diff
@@ -65,9 +65,9 @@ def __init__(self, cfg: Dict[str, any]):
         self.__load_config(cfg)
         self.__parse()
 
-    def __read_from_file(self, file: str):
-        with open(file) as f:
-            return yaml.load(f, Loader=yaml.FullLoader)
+    def __read_from_file(self):
+        with open(self.config_file) as f:
```

Comment:
Once https://github.com/Anon-Artist/arewefastyet/pull/2 is merged (which you can safely merge regardless of the failing GitHub Actions build), could you change on line 68: `def __read_from_file(self):` to `def __read_from_file(self, file):`. And on line 69, use the variable `file` instead of `self.config_file`.

Reply:
> Once [Anon-Artist#2](https://github.com/Anon-Artist/arewefastyet/pull/2) is merged (which you can safely merge regardless of the failing GitHub Actions build), could you change on line 68: `def __read_from_file(self):` to `def __read_from_file(self, file):`. And on line 69, use the variable `file` instead of `self.config_file`.

@Anon-Artist You just need to do the following to get the PR approved